packages/api/api-generator/src/commands/generate/generateFiles.ts

```diff
@@ -61,7 +61,5 @@ export const generateFiles = async (
                 }
             }
         }
-
-        await orm.close(true);
```

Comment:
Can we remove this? Shouldn't we close the ORM somewhere?

Reply:
in demo this close suddenly took 1 minute (did not before). And as we don't open a connection to a real database I
don't see any need for a close, especially as the script exits anyway right after the close.